packages/ui/src/containers/PasscodeValidation/use-register-with-sms-passcode-validation.ts

```diff
@@ -23,16 +23,16 @@ const useRegisterWithSmsPasscodeValidation = (phone: string, errorCallback?: ()
   const { errorMessage, clearErrorMessage, sharedErrorHandlers } = useSharedErrorHandler();
   const { signInMode } = useSieMethods();
 
-  const { run: signInWithSmsAsync } = useApi(signInWithSms);
+  const requiredProfileErrorHandlers = useRequiredProfileErrorHandler(true);
```

Comment:
are these files in the refactor plan?

Reply:
yep, there won't have those multiple apis